packages/dns/dns.0.15.1/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "1.2"
+maintainer: "anil@recoil.org"
+authors: [
+  "Anil Madhavapeddy"
+  "Tim Deegan"
+  "Richard Mortier"
+  "Haris Rotsos"
+  "David Sheets"
+  "Thomas Gazagnaire"
+  "Luke Dunstan"
+]
+homepage: "https://github.com/mirage/ocaml-dns"
+bug-reports: "https://github.com/mirage/ocaml-dns/issues"
+license: "
+LGPL-2.0 &
+   LGPL-2.1 with OCaml linking exception &
+   ISC
+         "
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+dev-repo: "https://github.com/mirage/ocaml-dns.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix "--%{base-unix:enable}%-lwt" "--%{mirage-types:enable}%-mirage"]
```

Comment:
Actually the dependency is not needed, that line https://github.com/mirage/ocaml-dns/blob/master/_oasis#L59 is very suspicious